### PR TITLE
corrected ordering on target reports for groups

### PR DESCRIPTION
**Pull request by dsnyder-fairway**

### CODE REVIEW

**Comment by dsnyder-fairway** [comment]
Didn't have correct test data until script was posted in Slack 😃

File: webapp/src/main/webapp/src/app/assessments/results/view/target-report/target-report.component.ts [hunk 1]
```diff
@@ -176,6 +180,10 @@ export class TargetReportComponent implements OnInit, ExportResults {
       new Column({ id: 'standard-met-relative-residual-level', headerInfo: true })
     ];
 
+    this._orderingByColumnField[ 'claim' ] = this.createOrdering('claimOrder');
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/assessments/results/view/target-report/target-report.component.ts]: Should this be `claimOrder`?

Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/app/assessments/results/view/target-report/target-report.component.ts]: I thought it should have been `claimOrder` too, but it was giving me some inconsistent behavior until mapping `claim` to `claimOrder`

Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/app/assessments/results/view/target-report/target-report.component.ts]: @jtreuting I'll give that a shot

Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/app/assessments/results/view/target-report/target-report.component.ts]: @jtreuting @wtritch I removed `claimOrder` and `sortField`

File: webapp/src/main/webapp/src/app/assessments/results/view/target-report/target-report.component.ts [hunk 2]
```diff
@@ -143,6 +145,8 @@ export class TargetReportComponent implements OnInit, ExportResults {
   private _sessions: any[];
   private _filterBy: FilterBy;
   private _filterBySubscription: Subscription;
+  private _orderingByColumnField: { [ key: string ]: Ordering<AggregateTargetScoreRow> } = {};
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/assessments/results/view/target-report/target-report.component.ts]: Recommend re-naming this to `_orderingByIdentityField` for clarity

**Pull request merged**